.github/workflows/docs-pages.yaml

```
@@ -6,6 +6,7 @@ on:
   push:
     branches:
     - scylla-3.x
+    - 'branch-**'
```

What's that logic behind this one?

---

@roydahan Good catch! In this repo the version branches are named `scylla-` and not `branch-`.

```suggestion
    - 'scylla-**'
```